jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2330,6 +2330,16 @@ properties:
             type: string
             enum: [exptime, ivm]
             fits_keyword: RESWHT
+          pixfrac:
```

Comment:
I don't see `pixfrac` getting populated anywhere in the `resample` code

Reply:
doh!  Will fix.